k8s/ietfweb/secrets.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secrets-env
+type: Opaque
+stringData:
+  IETFWWW_ADMINS: |-
+    Robert Sparks <rjsparks@nostrum.com>
+    Kesara Rathnayake <kesara@staff.ietf.org>
+
+  IETFWWW_ALLOWED_HOSTS: ".ietf.org"  # newline-separated list also allowed
+  WAGTAILADMIN_BASE_URL: "https://www.ietf.org"
+
+  # Outgoing email details
+  IETFWWW_EMAIL_HOST: "www.mr.ietf.org"
+  IETFWWW_EMAIL_PORT: "10027"
```

Comment:
Even though its IP restricted, I would avoid exposing this.

Reply:
I'll set it to null in both `ietfweb` and `iabweb`.